validator/validator.protoascii

```diff
@@ -1489,7 +1487,7 @@ tags: {
   attrs: {
     name: "src"
     mandatory: true
-    value_regex: "https://cdn\\.ampproject\\.org/v\\d+\\.js"
+    value: "https://cdn.ampproject.org/v0.js"
```

Comment:
We have a follow up CL for that which splits up the script tag into multiple to address #875.